### PR TITLE
Updated LUKSO Relayer API page

### PR DESCRIPTION
**Pull request by mustafademiray**
- Added a few diagrams to visualize the flow
- Restructured the menu to improve readability

Page structure:
image

### CODE REVIEW

**Review by JeneaVranceanu — APPROVED** [review]
LGTM

File: docs/tools/services/relayer-developer.md [hunk 1]
```diff
@@ -11,154 +11,187 @@ This feature is currently in private beta. If you are interested, please [contac
...
-// example value
-0x00006f357c6a0020820464ddfac1bec070cc14a8daf04129871d458f2ca94368aae8391311af6361696670733a2f2f516d597231564a4c776572673670456f73636468564775676f3339706136727963455a4c6a7452504466573834554178
+deployUniversalProfileWithSalt();
 ```
 
-ℹ️ More info on [`VerifiableURI`](https://github.com/lukso-network/LIPs/blob/main/LSPs/LSP-2-ERC725YJSONSchema.md#verifiableuri)
+### 2. Register Existing Universal Profiles
 
-</details>
+You can register existing Universal Profiles with the Relayer to enable gasless transactions for your users.
```
Comment (CJ42) [inline docs/tools/services/relayer-developer.md:148]: Suggest also mentioning that the user will have a certain quota amount (20 Millions per month)

File: docs/tools/services/relayer-developer.md [hunk 2]
```diff
@@ -11,154 +11,187 @@ This feature is currently in private beta. If you are interested, please [contac
...
 
-</details>
+registerUniversalProfile();
+```
 
-Please refer to the [Swagger](https://relayer-api.testnet.lukso.network/docs#/External%20Api%20Endpoints/UniversalProfileController_deployUniversalProfile) docs for more information.
+## Integration Guide
 
-### Register Users and Universal Profiles
+1. Obtain API credentials by filling out the access request form.
```
Comment (CJ42) [inline docs/tools/services/relayer-developer.md:182]: Need a link to the form for developers.

Also this form link should be accessible and visible from the top.

```suggestion
1. Obtain API credentials by filling out the access [request form](https:///link-to-form.com).
```

File: docs/tools/services/relayer-developer.md [hunk 3]
```diff
@@ -11,154 +11,187 @@ This feature is currently in private beta. If you are interested, please [contac
...
 
-  - If the UP is not already registered, the relayer creates a new `User` with the new UP attached.
-  - New `User` is granted the default gas quota (20m).
+- For technical issues or questions, contact our support team at [support@lukso.network](mailto:support@lukso.network).
+- Join our [Discord community](https://discord.com/invite/lukso) for discussions and updates.
+- Explore the [LUKSO documentation](https://docs.lukso.tech/) for more information on building with Universal Profiles.
 
-- Developer may also provide a `User ID`, to register a Universal Profile for an already created User.
-  - For example a Developer may build a platform where enterprise users can deploy Universal Profiles for their organisation, we can attach multiple Universal Profiles per organisation
+By leveraging the LUKSO Relayer API, you can create seamless, user-friendly experiences for your decentralized applications on the LUKSO network.
```
Comment (CJ42) [inline docs/tools/services/relayer-developer.md:197]: Suggest moving all of this content to the top

**Review by CJ42 — COMMENTED** [review]
Looks good, very nice mermaid charts! 🫶🏻
Added some review comments

**Pull request merged**